File: 1572-matrix-diagonal-sum/1572-matrix-diagonal-sum.java
class Solution 
{
    public int diagonalSum(int[][] mat) 
    {
        int a = 0, b = 0;
        for (int i = 0; i < mat.length; i++)
        {
            for (int j = 0; j < mat.length; j++)
            {
                if (i == j)
                {
                    a += mat[i][j];
                }
                else if ((i + j == mat.length - 1) && (i != j))
                {
                    b += mat[i][j];
                }
            }
        }
        return (a + b);
    }
}

File: 1572-matrix-diagonal-sum/1572-matrix-diagonal-sum.py
class Solution:
    def diagonalSum(self, mat):
        a = 0
        b = 0

        for i in range(len(mat)):
            for j in range(len(mat)):
                if i == j:
                    a += mat[i][j]
                elif (i + j == len(mat) - 1) and (i != j):
                    b += mat[i][j]

        return a + b